src/modules/Providers/views/details/tabs/VirtualMachines/utils/helpers/toVmFeatureEnum.ts

```diff
@@ -1,6 +1,7 @@
+import type { TFunction } from 'react-i18next';
 import type { VmFeatures } from 'src/utils/types';
 
-export const toVmFeatureEnum = (t: (string) => string): { [k in keyof VmFeatures]: string } => ({
+export const toVmFeatureEnum = (t: TFunction): { [k in keyof VmFeatures]: string } => ({
```

Comment:
I think it is easier instead of adding a type, simply import t directly here and omit it from args